plugins/system/shortcut/shortcut.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * @package     Joomla.Plugins
+ * @subpackage  System.shortcut
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Plugin\CMSPlugin;
+/**
+ * Shortcut plugin to add accessible keyboard navigation to the site and administrator templates.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class PlgSystemShortcut extends CMSPlugin
+{
+	/**
+	 * Application object.
+	 *
+	 * @var    JApplicationCms
```

Comment:
Use named space name `\Joomla\CMS\Application\CMSApplication` and in this case please use the interface `CMSWebApplicationInterface`